server/src/main/java/org/elasticsearch/Build.java

```diff
@@ -130,6 +130,82 @@ public static String minimumCompatString(IndexVersion minimumCompatible) {
         }
     }
 
+    /**
+     * Check whether a version is compatible with the current minimum transport version.
+     * @param version A version identifier as a string
+     * @throws IllegalArgumentException if version is not a valid transport version identifier
+     * @return true if the version is compatible, false otherwise
+     */
+    public static boolean isWireCompatibleWithCurrent(String version) {
+        try {
+            TransportVersion transportVersion = TransportVersion.fromString(version);
+            return transportVersion.onOrAfter(TransportVersions.MINIMUM_COMPATIBLE);
+        } catch (NumberFormatException | IllegalStateException e) {
```

Comment:
Where does the IllegalStateException come from?

Reply:
I'm not sure, but I don't think we actually need this method so it's been removed anyway.